guides/release/routing/query-params.md

```diff
@@ -83,25 +83,26 @@ With this code, we have established the following behaviors:
    full router transition (i.e. it won't call `model` hooks and
    `setupController`, etc.); it will only update the URL.
 
-### link-to Helper
+### LinkTo component
```

Comment:
I think we probably want `<LinkTo />` here to be consistent, but I'm not 100% sure 🤔